geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIdxStrategy.scala

```diff
@@ -72,17 +65,17 @@ class AttributeIdxStrategy(val filter: QueryFilter) extends Strategy with LazyLo
     val priority = FILTERING_ITER_PRIORITY
 
     // query against the attribute table
-    val singleAttrValueOnlyPlan: ScanPlanFn = (schema, filter, transform) => {
-      val iters = KryoLazyFilterTransformIterator.configure(schema, filter, transform, sampling).toSeq
+    val singleAttrValueOnlyPlan: ScanPlanFn = (schema, ecql, transform) => {
```

Comment:
filter in this case is the QueryFilter from line 29 - I changed the name here to avoid variable name overlap.